content/Foundation/Summit/2023/cfp.pml

```diff
@@ -202,23 +190,16 @@ We solicit proposals on any seL4-related topic, in particular in the areas of:
   If you would like to propose a talk (or nominate someone else to be invited to
   do so), please upload an abstract of <strong>one page or less</strong> for your
   proposed talk (or a one paragraph description of your nominee’s proposed talk)
-  <strong>by [Date TBC]</strong> to ..[link to submission portal]. Abstracts and
+  <strong>by 31 March 2023</strong> to ..<strong>[link to submission portal]</strong>. Abstracts and
```

Comment:
- date to be updated after discussion
- "link to submission" to be changed to "to the submission portal (opening soon)."